src/dependency-builder/build-tree.js

```diff
@@ -228,6 +188,24 @@ function resolveModulePath(nmPath, workingDir, root) {
   return resolveModulePath(nmPath, parentWorkingDir, root);
 }
 
+/**
+ * Resolve package dependencies from package.json
+ *
+ * @param {string} cwd
+ * @param {string []} missing
+ * @param {string} consumerPath
+ * @returns new object with found and missing
+ */
+function resolvePackageDependeces (cwd: string, missing: string[] ) {
+  const { dependencies, devDependencies, peerDependencies } = PackageJson.findPackage(cwd);
+  const mergedDependencies = Object.assign(dependencies, devDependencies, peerDependencies)
```

Comment:
what happen if there is no devDependencies / peerDependencies make sure it's not throw an error